tutorials/RandomNumberGeneration/README.md

```diff
@@ -0,0 +1,2 @@
+# Welcome!
+
```

Comment:
Here I would add a sentence or two that would repeat the first paragraph of the tutorial (you'll quantum random number generation, which is very simple but potentially useful application, etc.), and brief instructions on running the tutorial (similar to https://github.com/microsoft/QuantumKatas/blob/master/tutorials/ExploringDeutschJozsaAlgorithm/README.md - it's not designed to be used with C# project)